devicetypes/smartthings/springs-window-fashions-shade.src/springs-window-fashions-shade.groovy

```diff
@@ -266,4 +261,29 @@ def refresh() {
             zwave.switchMultilevelV1.switchMultilevelGet().format(),
             zwave.batteryV1.batteryGet().format()
     ], 1500)
+}
+
+def levelChangeFollowUp(expectedLevel) {
+    state.expectedValue = expectedLevel
```

Comment:
This is more of a nit-picking, but `state.expectedLevel` would be more clear than `state.expectedValue`.

Reply:
I still think `state.expectedValue` should be `state.expectedLevel`.